EosPayload/drivers/led_driver.py

```diff
@@ -3,29 +3,42 @@
 
 from EosPayload.lib.base_drivers.driver_base import DriverBase
 
+""" NOT IMPLEMENTED YET
+LED_1 -> Red     | GPS Fix
+LED_2 -> Orange  | Running
+LED_3 -> Green   | Transmitting
+"""
+
 
 class LEDDriver(DriverBase):
 
     def __init__(self, output_directory: str, config: dict):
         super().__init__(output_directory, config)
-        self.pin_name = "P9_12"
+        self.led_1 = "P9_21"
+        self.led_2 = "P9_23"
+        self.led_3 = "P9_25"
```

Comment:
this should really be the last time we hardcode pins like this, going forward we should define this in the config file

Reply:
Personally, I think it's safe to hardcode because the only time they would change would be if you have a new PCB

Reply:
True, but can this be a #laterproblem I need to work on stuff :)